pkgs/development/libraries/libbladeRF/default.nix

```
@@ -21,6 +21,15 @@ stdenv.mkDerivation rec {
   # Fixup shebang
   prePatch = "patchShebangs host/utilities/bladeRF-cli/src/cmd/doc/generate.bash";
 
+  # Fixes macos and freebsd compilation issue.
+  # https://github.com/Nuand/bladeRF/commit/0cb4ea888543b2dc75b876f7024e180854fbe9c3
+  patches = [ (fetchpatch {
+                name = "fix-OSX-and-FreeBSD-build.patch";
+                url = "https://github.com/Nuand/bladeRF/commit/0cb4ea88.diff";
+                sha256 = "1ccpa69vz2nlpdnxprh4rd1pgphk82z5lfmbrfdkn7srw6nxl469";
+              })
+            ];
+
   # Let us avoid nettools as a dependency.
   patchPhase = ''
```

This one overrides the patch phase. Thus the patch is never really applied. ``postPatch`` is the better choice here:
```suggestion
  postPatch= ''
```

---

@greydot Could you handle this comment? It should (hopefully) fix the darwin build :)

---

@Ekleog done. Sorry for the delay.